src/metric.jl

```diff
@@ -179,6 +179,47 @@ function reset!(metric :: MSE)
   metric.n_sample = 0
 end
 
+doc"""
+    NMSE
+
+Normalized Mean Squared Error
+
+```math
+\sum_i (\frac{label_i - pred_i}{label_i})^2
```

Comment:
No worries and excellent explanation. If other cases popup we can introduce different cases of this.